sl/symheap.cc

```diff
@@ -808,7 +808,7 @@ bool SymHeapCore::Private::chkValueDeps(const TValId val)
         return true;
 
     const InternalCustomValue *customData =
-        dynamic_cast<const InternalCustomValue *>(valData);
+        DCAST<const InternalCustomValue *>(valData);
```

Comment:
This is not necessary. This function runs only in debug mode -> always as `dynamic_cast`.

Reply:
Good point.  I have pushed a fixup commit to revert this change.